docs/HTML.md

```diff
@@ -0,0 +1,203 @@
+### 1. `<hr>` -- horizontal rule (dividing line)
+
+* *type:* block-level
+* *content:* none, empty element
+* *support:* all browsers
+* *example:*
+```
+<p>some text</p>
+<hr>
+<p>text down under</p>
+```
+<p>some text</p>
+<hr>
+<p>text down under</p>
+
+### 2. `<table>` -- HTML Table element represents data in 2 dimensions or more
+
+* *type:* block-level
+* *content:* Flow Content[1]
+* *support:* all browsers
+* *example:*
+```
+<table>
+<tr>
+<td>top left</td>
+<td>top right</td>
+</tr>
+<tr>
+<td>bottom left</td>
+<td>bottom right</td>
+</tr>
+</table>
+```
+<table>
+<tr>
+<td>top left</td>
+<td>top right</td>
+</tr>
+<tr>
+<td>bottom left</td>
+<td>bottom right</td>
+</tr>
+</table>
+
+### 3. `<pre>` -- Preformatted Text, observes whitespace
+
+* *type:* block-level
+* *content:* phrasing content[2]
+* *support:* all browsers
+* *example:*
+```
+<pre>
+derp      space
+s p a c e d
+</pre>
+```
+<pre>
+derp      space
+s p a c e d
+</pre>
+
+### 4. `<dl>` -- Description list element
+
+* *type:* block-level
+* *content:* Flow content[1]
+* *support:* all browsers
+* *example:*
+```
+<dl>
+<dt>Jon</dt>
+<dd>Some guy that signed up for the iron yard</dd>
+</dl>
+```
+<dl>
+<dt>Jon</dt>
+<dd>Some guy that signed up for the iron yard</dd>
+</dl>
+
```

Comment:
This is another member of a "family" of elements like `<ul>`, `<ol>` and `<li>` that really should be documented together to get the full use context.